crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -90,12 +90,21 @@ def adminrepocheck
           if zypper_stream["message"] =~ /^System management is locked/
             return {
               status: :service_unavailable,
-              message: I18n.t(
+              error: I18n.t(
```

Comment:
OK, maybe not. I meant the result of admin repo check when the repos are actually not there, but we do not have to treat it like an error, just a special kind of result.